extensions/typescript-language-features/src/tsServer/bufferSyncSupport.ts

```diff
@@ -684,6 +684,8 @@ export default class BufferSyncSupport extends Disposable {
 	}
 
 	private triggerDiagnostics(delay: number = 200) {
+		/// MEMBRNAE: make intellisense and ts-plugin respond faster by reducing the artificial delay.
```

Comment:
Is triple slash a convention you're using for membrane comments? or just muscle memory from rust

Reply:
The latter, whoops